student.rb

```diff
@@ -1,12 +1,19 @@
-require './person'
+require_relative 'person'
 
 class Student < Person
-  def initialize(age, classroom, _name = 'Unknown', parent_permission: true)
+  attr_accessor :classroom
```

Comment:
- [ ] Good job so far on the association but you just need to fix some issues and you are good to go, the `classroom` is a `setter` which means the `classroom` should be `attr_reader`. Kindly fix that 🙏🏾

Reply:
Thank you for noticing, It's done!